docs/src/_cloud/connectors.md

```diff
@@ -62,3 +62,29 @@ There are multiple ways to improve a connector's quality. Please [review this pa
 **Can I request a connector to be built or improved if it doesn’t meet my needs?**
 
 Yes. Please [reach out to us](https://meltano.com/contact/) with your request.
+
+## Platinum Connector Support
+
+Meltano offers Platinum Connectors Support for many connectors when run on Meltano Cloud.
+Platinum support is included with all tiers of Meltano Cloud.
+
+To learn more about which connectors are Platinum Certified, please [reach out to the team](https://meltano.com/contact/) or contact your account representative if you're a current Cloud customer.
+
+### Platinum Certified Connector - Guarantees
+
+Each Platinum Connector will have the following guarantees:
+
+* Built with the Meltano SDK
+* Open Source License
+* Semantic Versioning used in all releases
+* Daily integration testing
```

Comment:
What will handle this? Do we have an issue open for this?

Reply:
@WillDaSilva I was planning to bring this to the Singer Guild. Meltano Cloud is the solution here. We could have specific projects for these connectors running to any number of targets (duckdb at first or even snowflake) along with specific dbt models doing tests to validate the data. 

Adds a nice benefit for drinking our own Cloud champagne _and_ for regular testing of these first-party connectors.

Reply:
@tayloramurphy Do we have any issues open to track this? If we do, they may impact:
- https://github.com/meltano/meltano/issues/2566

Reply:
I started https://github.com/meltano/internal-product/discussions/244